apps/ledger-live-mobile/src/screens/Settings/Debug/Features/BLEPairingFlow.tsx

```diff
@@ -60,7 +60,7 @@ export default () => {
     pairedDevice: null,
   };
 
-  const promptBluetooth = usePromptBluetoothCallback();
+  const promptBluetooth = usePromptEnableBluetoothCallback();
 
   const goToBlePairingFlow = useCallback(() => {
```

Comment:
related to the code below inside the callback, which actually didn't change, but maybe should:

if we first prompt the enabling of Bluetooth _before_ navigating to the bluetooth  device pairing screen, aren't we creating the impression, for a person checking that debugging screen, that the screen itself is handling this enabling of bluetooth?
That screen actually does it, but imagine it breaks in that screen / gets removed, then it won't be detected here, because in any case we try to enable Bluetooth here before actually navigating to the screen (which is supposed to do that too).

Reply:
Especially, now we request if the component `RequiresBLE` is used (as it should be). So if the user refuses once on the prompt coming from this ` const promptBluetooth = usePromptEnableBluetoothCallback();`, then it is again prompt by the `RequiresBLE` just after